github/orgs_teams.go

```diff
@@ -46,17 +46,46 @@ func (t Team) String() string {
 
 // Invitation represents a team member's inviation status
 type Invitation struct {
-	ID        *int       `json:"id,omitempty"`
-	Login     *string    `json:"login,omitempty"`
-	Email     *string    `json:"email,omitempty"`
-	Role      *string    `json:"role,omitempty"`
-	CreatedAt *time.Time `json:"created_at,omitempty"`
+	ID    *int    `json:"id,omitempty"`
+	Login *string `json:"login,omitempty"`
+	Email *string `json:"email,omitempty"`
+	Role  *string `json:"role,omitempty"`
+	//golang documentation states
+	//Programs using times should typically store and pass them as values, not pointers.
+	//That is, time variables and struct fields should be of type time.Time, not *time.Time
```

Comment:
Okay, question, have you considered using `*Timestamp` here instead of `*time.Time`?

Can you be more specific about the problem you're seeing with `*time.Time` as is?